tests/test_matplotlibFormatter.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for MatplotlibFormatter.
+"""
+
+import unittest
+import tempfile
+import os
+import shutil
+
+import matplotlib
```

Comment:
Is this resolved?  I agree that it's important.

Reply:
(It _looks_ resolved, given the new `try` block for the import of `matplotlib`, but I haven't been able to test it in practice.)

Reply:
It's hard for any of us to test it, I think, because we all use conda enviroments that always need to have matplotlib for other reasons.  But I'm pretty confident what was merged will work.